src/vs/monaco.d.ts

```diff
@@ -4133,6 +4148,9 @@ declare namespace monaco.editor {
 		showFoldingControls: IEditorOption<EditorOption.showFoldingControls, 'always' | 'mouseover'>;
 		showUnused: IEditorOption<EditorOption.showUnused, boolean>;
 		showDeprecated: IEditorOption<EditorOption.showDeprecated, boolean>;
+		showInlineHints: IEditorOption<EditorOption.showInlineHints, boolean>;
```

Comment:
Maybe have a config object like `editor.inlineHints.enabled`, `editor.inlineHints.fontSize` etc. That pattern is becoming more popular but I'd need to check myself what the recommendation is.